frontend/src/components/maps/PopupView.tsx

```diff
@@ -14,11 +14,17 @@ export const PopupView: React.FC<IPopupViewProps> = ({
   propertyDetail,
   disabled,
 }) => {
-  if (propertyTypeId === 0) {
+  if (propertyTypeId === PropertyTypes.PARCEL) {
     return <ParcelPopupView disabled={disabled} parcel={propertyDetail as IParcel} />;
   }
-  if (propertyTypeId === 1) {
+  if (propertyTypeId === PropertyTypes.BUILDING) {
     return <BuildingPopupView building={propertyDetail as IBuilding} />;
   }
+  if (propertyTypeId === PropertyTypes.DRAFT_PARCEL) {
```

Comment:
This just helps the user to remember which pin is which (usually an issue if there are a lot of buildings).

Reply:
A draft is before it's submitted to inventory?

Reply:
yes, draft markers are markers that have been updated in the form but not saved yet.